libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -515,3 +519,34 @@ void AP_L1_Control::update_level_flight(void)
 
     _data_is_stale = false; // status are correctly updated with current waypoint data
 }
+
+// update L1 control for path following
+void AP_L1_Control::update_path(const class Location &position_on_path, Vector2f unit_path_tangent, float path_curvature, int8_t direction) {
+    //! @note initial implementation uses existing functions
+    if (!is_zero(path_curvature)) {
+        // moving along arc of circle - loiter about wp located at
+        // centre of curvature.
+        float radius_m = 1.0 / path_curvature;
```

Comment:
fabs() ?

Reply:
The curvature is computed as `path_curvature = accel_lat.length() / vel.length_squared();` which is always positive.
Should we add a comment that curvature is assumed to be positive in the header?